src/main/resources/db/changelog/db.changelog.xml

```diff
@@ -538,4 +538,9 @@
         </createTable>
     </changeSet>
 
+    <changeSet id="045-add-moderation-required-column-to-blog-table" author="vlosrutere">
```

Comment:
I would make this column not nullable and also update existing records to false/true as described in one of the comments above.